pkg/gallery/export.go

```diff
@@ -15,6 +15,7 @@ func ToBasicJSON(gallery *models.Gallery) (*jsonschema.Gallery, error) {
 	newGalleryJSON := jsonschema.Gallery{
 		Title:     gallery.Title,
 		URL:       gallery.URL,
+		Date:      gallery.Date.Time.String(),
```

Comment:
This is already covered in the if statement below. This will panic if `Date` is `nil`.

Reply:
Done